apps/web/components/molecules/commentItem/commentItem.tsx

```diff
@@ -1,34 +1,29 @@
-import Image from 'next/image';
 import { Card } from 'ui';
 import { getTimeDiffFromTimestamp } from 'utils/timeDifference/getTimeDiffFromTimestamp';
 import type { Comment } from '../../../types/types';
 import { ReactComponent as Reply } from '/components/icons/reply.svg';
 import { ReactComponent as Like } from '/components/icons/like.svg';
+import { CommentReply } from 'molecules/commentItem/commentReply';
+import { UserAvatar } from 'molecules/userAvatar/userAvatar';
 
 type CommentItemProps = Comment;
 
 export const CommentItem = ({
 	author,
 	avatar,
 	comment,
-	date,
+	timestamp,
 }: CommentItemProps) => {
```

Comment:
I'd make a `comment: Comment` prop accepting an object. No need for spreading then and you have everything contained in one object

Reply:
> 



> I'd make a `comment: Comment` prop accepting an object. No need for spreading then and you have everything contained in one object

Sure, but you neeed to destructure it later or write redundant object prop calls

```
comment.timestamp
comment.avatar
...and so on
```

Reply:
I'd rather have a param destructurization than something like that tbh.
As for now the `CommentItemProps` is redundant as the props could be directly typed as `Comment`.
I used to spread props but over the time I learned in most of the cases it is better to explicitly pass props. There are some exceptions ofc (for example `register` function for form libs)